presto-main/src/test/java/com/facebook/presto/operator/scalar/TestKleinbergBurstDetectionFunction.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.scalar;
+
+import com.facebook.presto.spi.type.ArrayType;
+import com.facebook.presto.spi.type.IntegerType;
+import com.google.common.collect.ImmutableList;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.spi.StandardErrorCode.INVALID_FUNCTION_ARGUMENT;
+
+public class TestKleinbergBurstDetectionFunction
+        extends AbstractTestFunctions
+{
+    @Test
+    public void testKleinbergBurstDetection()
+    {
+        // Input with one entry.
+        assertInvalidFunction(
+                "KLEINBERG_BURST_DETECTION(ARRAY[1.1])",
+                INVALID_FUNCTION_ARGUMENT);
```

Comment:
Validate the error message, to make sure we're testing the correct error condition